src/aggregator/aggregator/follower_flush_mgr_test.go

```diff
@@ -131,6 +132,56 @@ func TestFollowerFlushManagerCanNotLeadForwardedFlushWindowsNotEnded(t *testing.
 	require.False(t, mgr.CanLead())
 }
 
+func TestFollowerFlushManagerCanLeadNotFlushed(t *testing.T) {
```

Comment:
Can we add some test cases around window boundaries? Things like:

```
manager opened right after window start
manager opened right before window start
```
etc.

Reply:
Both cases in the bottom of this test.